### DISCUSSION TITLE
Trying to reset admin password

### DISCUSSION THREAD

**Discussion by Neo7252** [requarks/wiki#6941] [Help / Questions] [answered]
Hey,

I lost my admin password. I'm trying to reset it using Docker command line and update the database ([Link](https://docs.requarks.io/en/troubleshooting#how-to-manually-reset-the-admin-password)).

`psql -U wiki -d wikij -c "UPDATE users SET password = '?pjzAHbL4roDePpL' WHERE email = 'MyEmail@gmail.com';"`

But I have this error:
psql: error: connection to server on socket "/var/run/postgresql/.s.PGSQL.5432" failed: FATAL:  role "wiki" does not exist

The first 'wiki' doesn't exist, what is this parameter?

Thanks!

**Comment by fnafox** [comment, marked as answer]
Have you changed the db user or database variables when setting up the wiki?
Check them in the wiki container:
```
docker exec wiki echo "$DB_USER"
docker exec wiki echo "$DB_NAME"
```

**Reply by Neo7252** [reply]
I have the correct information now, thanks!
